src/query/api/v1/handler/fetch_options_test.go

```diff
@@ -103,6 +115,25 @@ func TestFetchOptionsBuilder(t *testing.T) {
 			},
 			expectedErr: true,
 		},
+		{
+			name:  "can set lookback duration",
+			query: "lookback=10s",
+			expectedLookback: &expectedLookback{
+				value: 10 * time.Second,
+			},
+		},
+		{
+			name:  "can set lookback duration based on step",
+			query: "lookback=step&step=10s",
```

Comment:
May be worth adding in `lookback=step`, `lookback=step&step=invalid` , `lookback=step&step=-1` cases here?

Reply:
Sure thing.